.eslintrc.json

```diff
@@ -6,6 +6,7 @@
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
     "plugin:prettier/recommended"
```

Comment:
Remove `plugin:prettier/recommended`, add just `prettier` here. Prettier recommends not to use the plugin anymore.